rest/data/interface.go

```diff
@@ -128,5 +129,13 @@ type Connector interface {
 	SetPatchPriority(string, int64) error
 	SetPatchActivated(string, string, bool) error
 
+	// GetAdminSettings/SetAdminSettings retrieves/sets the system-wide settings document
+	GetAdminSettings() (*admin.AdminSettings, error)
+	SetAdminSettings(*admin.AdminSettings, user.DBUser) error
```

Comment:
MustHaveUser() returns a *user.DBUser pointer rather than a concrete struct. 

I think we should probably be consistent here.